@pauliescanlon/gatsby-mdx-embed/src/components/CodePen/CodePen.tsx

```diff
@@ -9,11 +9,11 @@ export interface ICodePenProps {
   tabs?: string[] | 'js' | 'css' | 'scss' | 'less' | 'result'
 }
 
-export const CodePen: FunctionComponent<ICodePenProps> = ({
+export const CodePen: FunctionComponent<Props> = ({
   codePenId,
   height = 500,
   tabs = 'result'
-}: ICodePenProps) => (
```

Comment:
Is it not best practice to type the return here?

Reply:
You're already typing the variable (the component), so this wil implicitly type the props object for you correctly.

Reply:
Thats good though isn't it. It means the component and its props are more strictly typed... or am i confusing the two things here?

Reply:
Because you're already typing the variable (that's the way how it should be), you don't have to explicitly type the props anymore.